src/js/node/fs.ts

```diff
@@ -11,6 +14,10 @@ const isDate = types.isDate;
 const { fs } = promises.$data;
 
 const constants = $processBindingConstants.fs;
+var _lazyGlob;
```

Comment:
this wrapper is not needed. it is ok to inline this require call in the two call sites

Reply:
this comment was not addressed.

Reply:
It was that way originally, but I changed it due to @nektro's feedback. I'm getting conflicting signals :L